Author: Daniel Baumann <daniel.baumann@progress-linux.org>
Description: Avoid removing files after build.

diff -Naurp twitter-bootstrap.orig/Makefile twitter-bootstrap/Makefile
--- twitter-bootstrap.orig/Makefile	2012-02-10 21:28:22.127338647 +0100
+++ twitter-bootstrap/Makefile	2012-02-10 21:28:11.079284627 +0100
@@ -11,7 +11,6 @@ WATCHR ?= `which watchr`
 
 docs: bootstrap
 	zip -r docs/assets/bootstrap.zip bootstrap
-	rm -r bootstrap
 	lessc ${BOOTSTRAP_LESS} > ${BOOTSTRAP}
 	lessc ${BOOTSTRAP_RESPONSIVE_LESS} > ${BOOTSTRAP_RESPONSIVE}
 	node docs/build
@@ -24,6 +23,7 @@ docs: bootstrap
 #
 
 bootstrap:
+	rm -rf bootstrap
 	mkdir -p bootstrap/img
 	mkdir -p bootstrap/css
 	mkdir -p bootstrap/js
