From: Markus Koschany <apo@gambaru.de>
Date: Sat, 8 Jun 2013 23:05:57 +0200
Subject: use nodejs command

---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5a31595..b69fc88 100644
--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@ docs: bootstrap
 	zip -r docs/assets/bootstrap.zip bootstrap
 	lessc ${BOOTSTRAP_LESS} > ${BOOTSTRAP}
 	lessc ${BOOTSTRAP_RESPONSIVE_LESS} > ${BOOTSTRAP_RESPONSIVE}
-	node docs/build
+	nodejs docs/build
 	cp img/* docs/assets/img/
 	cp js/*.js docs/assets/js/
 
